model_zoo/spleen_ct_segmentation_monai_bundle/README.md

```diff
@@ -0,0 +1,78 @@
+# Description
+Adapt a bundle to another dataset
+
+# Overview
+This tutorial shows how to adapt an example monai bundle from MONAI model-zoo to a new dataset.
+
+## Data
+The new dataset is BTCV challenge dataset (https://www.synapse.org/#!Synapse:syn3193805/wiki/217752). It has 24 Training + 6 Validation CT abdominal scans.
+Introduction of BTCV dataset can be found in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb.
+
+There are 13 organs labeld in BTCV. In this example, we focus on spleen segmentation only.
+
+Step 1: Download BTCV dataset RawData.zip following the instruction in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb. Extract it as `./data/RawData`.
+
+Step 2: Download the the json file for data splits in https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb, and save it to `./data/dataset_0.json`.
+
+Step 3: The labels in BTCV dataset contains 13 organs. We split the labels and extract spleen label out. This is done by running `python ./split_spleen_labels.py`.
```

Comment:
I would suggest to change the file name `split_spleen_labels.py` to `split_labels.py`.
The dataset content is not spleen anymore.

Thanks.

Reply:
I did not use all the labels. Actually this script extract spleen out, so I could use the spleen model in the model-zoo on it. I did try an experiment using all 13 organs, but in that case, transfer learning did not help.

Reply:
Oh, I see. Then that makes sense to me.
Thanks for the explanation.

Reply:
Maybe you can enhance the README.md about how you extracted "only" the spleen data.

Thanks.

Reply:
Sure. I updated it to: 

Step 3: The segmentation labels in BTCV dataset contain 13 organs. Therefore there are 14 intensity levels in the label images.
In this experiment, we focus on spleen only, which has intensity = 1 in the label images. We split the labels and create binary spleen masks by running `python ./split_spleen_labels.py`.